components/filecoin/rs/src/signature.rs

```diff
@@ -1,55 +1,27 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
I've reviewed the changes in this file for cryptography usage and have no concerns with what is being done. This is using libraries we've already merged into place and the usage does not appear to introduce any cryptographic vulnerabilities.